README.md

```diff
@@ -0,0 +1,190 @@
+
+## Requirements
+
+The following requirements exist for running Apache JMeter:
+
+ -  Java Interpreter:
+
+    A fully compliant Java 7 (or later) Runtime Environment is required 
+    for Apache JMeter to execute.
+
+ -  Optional jars:
+
+    Some jars are not included with JMeter.
+    If required, these should be downloaded and placed in the lib directory
+
+    + JDBC - available from database supplier
+    + JMS - available from the JMS provider
+    + Bouncy Castle - available from http://www.bouncycastle.org/latest_releases.html
+
+      (currently only needed for SMIME Assertion)
+
+ -  Java Compiler [OPTIONAL]:
+
+    A Java compiler is not needed since the distribution includes a
+    precompiled Java binary archive. _Note that a compiler is required
+    if you plan to build plugins for Apache JMeter._
+
+## Installation Instructions
+
+_Note that spaces in directory names can cause problems._
+
+ - Release builds
+
+   Unpack the binary archive into a suitable directory structure.
+
+## Running JMeter
+
+Change to the bin directory
+Run the jmeter (Un\*x) or jmeter.bat (Windows) file.
+
+For Windows there are also some other scripts.
+Drag-drop a JMX file onto one of them:
+
+`jmeter-n.cmd` - runs the file as a non-GUI test
+
+`jmeter-n-r.cmd` - runs the file as a non-GUI remote (client-server) test
+
+`jmeter-t.cmd` - loads the file ready to run it as a GUI test
+
+## Documentation
+
+The documentation available as of the date of this release is
+also included, in HTML format, in the `printable_docs/` directory,
+and it may be browsed starting from the file called `index.html`.
+
+## Build instructions
+
+ - Release builds
+
+   Unpack the source archive into a suitable directory structure.
+   Most of the 3rd party library files can be extracted from the binary archive
+   by unpacking it into the same directory structure.
+   You can also use Ant to download the required library files:
+
+```sh
+ant download_jars
+```
+
+Any optional jars (see above) should be placed in `lib/opt` and/or `lib`.
+
+Jars in `lib/opt` will be used for building JMeter and running the unit test,
+but won't be used at run-time.
+
+_This is useful for testing what happens if the optional jars are not
+downloaded by other JMeter users._
+
+JMeter is built using Ant.
+
+Change to the top-level directory and issue the command:
+
+```sh
+ant download_jars
+```
+_Only needs to be done once; will download any missing 3rd party jars_
+
+```sh
+ant
+```
+
+This will compile the application and enable you to run jmeter from the `bin`
+directory.
+
+```sh
+ant test [-Djava.awt.headless=true]
+```
+
+This will compile and run the unit tests.
+The optional property definition is required if the system
+does not have a suitable GUI display.
+
+## Developer information
+
+The code is maintained in SVN at https://svn.apache.org/repos/asf/jmeter/trunk
+
+There is a read-only mirror at GitHub: https://github.com/apache/jmeter
```

Comment:
Might be a good idea. At the moment README.md is a direct translation of the old README file.